lerobot/common/robot_devices/robots/stretch.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python
+
+# Copyright 2024 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+from dataclasses import dataclass, field, replace
+
+import torch
+from stretch_body.gamepad_teleop import GamePadTeleop
+from stretch_body.robot import Robot as StretchAPI
+
+from lerobot.common.robot_devices.cameras.utils import Camera
+
+
+@dataclass
+class StretchRobotConfig:
+    robot_type: str | None = "stretch"
+    cameras: dict[str, Camera] = field(default_factory=lambda: {})
+    # TODO(aliberts): add feature with max_relative target
+    # TODO(aliberts): add comment on max_relative target
+    max_relative_target: list[float] | float | None = None
+
+
+class StretchRobot(StretchAPI):
+    """Wrapper of stretch_body.robot.Robot"""
+
+    def __init__(self, config: StretchRobotConfig | None = None, **kwargs):
+        super().__init__()
+        if config is None:
+            config = StretchRobotConfig()
+        # Overwrite config arguments using kwargs
+        self.config = replace(config, **kwargs)
+
+        self.robot_type = self.config.robot_type
+        self.cameras = self.config.cameras
+        self.is_connected = False
+        self.teleop = None
+        self.logs = {}
+        # TODO(aliberts): remove original low-level logging from stretch
```

Comment:
You can add

https://github.com/hello-robot/stretch_body/blob/5ef6564dcdd93fb8120e44ad4d34017617210142/tools/bin/stretch_configure_tool.py#L2-L4

at the top of the file to raise the logging level. I might be able to also add a boolean to outright disable Stretch Body logging if it still interferes with your loggers. Let me know.

Reply:
I've tried that in the past and it didn't work, but somehow since then the logs from Stretch have disappeared and I'm not exactly sure why 😅
I've added it https://github.com/huggingface/lerobot/pull/409/commits/e0fa5e43f320207f02df138820b09a9619bf559f

Reply:
That's interesting 😂. I'll let you know if I see them when I run it.